packages/utilities/psammead-locales/moment/ar.js

```diff
@@ -27,4 +27,11 @@ moment.updateLocale('ar', {
     // Suppress conversion to Eastern numerals from base `ar` locale but retain use of Arabic comma
     return string.replace(/,/g, '،');
   },
+  relativeTime: {
+    past: '%s',
+    m: 'قبل دقيقة واحدة',
```

Comment:
I believe moment expects to return a function here. 
https://github.com/moment/moment/blob/develop/src/locale/ar.js#L103
This is the what the value is for `m` in moment config:

```
m: function (number, withoutSuffix, string, isFuture) {
        var f = pluralForm(number),
            str = plurals['m'][pluralForm(number)];
        if (f === 2) {
            str = str[withoutSuffix ? 0 : 1];
        }
        return str.replace(/%d/i, number);
    };
```

Reply:
Ah, no worries, it seems like that is a special case. Other locales just return strings. https://github.com/moment/moment/blob/develop/src/locale/fa.js#L68